cmd/soci/commands/create.go

```diff
@@ -41,15 +45,19 @@ var CreateCommand = cli.Command{
 	ArgsUsage: "[flags] <image_ref>",
 	Flags: []cli.Flag{
 		cli.Int64Flag{
-			Name:  "span-size",
```

Comment:
why was this changed?

Reply:
I've redefined these flags as `const`s up top. I think it's cleaner than just having bare strings.